runtime/devnet/src/extensions/mod.rs

```diff
@@ -247,54 +187,24 @@ where
 	env.write(&result, false, None)
 }
 
-fn send_xcm<T, E>(env: &mut Environment<E, BufInBufOutState>) -> Result<(), DispatchError>
-where
-	T: pallet_contracts::Config
-		+ frame_system::Config<
-			RuntimeOrigin = RuntimeOrigin,
-			AccountId = AccountId,
-			RuntimeCall = RuntimeCall,
-		>,
-	E: Ext<T = T>,
-{
-	const LOG_PREFIX: &str = " send_xcm |";
-	// Read the input as CrossChainMessage.
-	let xc_call: CrossChainMessage = env.read_as::<CrossChainMessage>()?;
-	// Determine the call to dispatch.
-	let (dest, message) = match xc_call {
-		CrossChainMessage::Relay(message) => {
-			let dest = Location::parent().into_versioned();
-			let assets: Asset = (Here, 10 * UNIT).into();
-			let beneficiary: Location =
-				AccountId32 { id: (env.ext().address().clone()).into(), network: None }.into();
-			let message = Xcm::builder()
-				.withdraw_asset(assets.clone().into())
-				.buy_execution(assets.clone(), Unlimited)
-				.transact(
-					SovereignAccount,
-					Weight::from_parts(250_000_000, 10_000),
-					message.encode().into(),
-				)
-				.refund_surplus()
-				.deposit_asset(assets.into(), beneficiary)
-				.build();
-			(dest, message)
-		},
-	};
-	// TODO: revisit to replace with signed contract origin
-	let origin: RuntimeOrigin = RawOrigin::Root.into();
-	// Generate runtime call to dispatch.
-	let call = RuntimeCall::PolkadotXcm(pallet_xcm::Call::send {
-		dest: Box::new(dest),
-		message: Box::new(VersionedXcm::V4(message)),
-	});
-	dispatch_call::<T, E>(env, call, origin, LOG_PREFIX)
+// Example wrapper to enable versioning of `RuntimeStateKeys`.
```

Comment:
Remove `example` in comment, convert to doc comments and add doc comments for variants if this is being retained.

Reply:
see refactor: devnet runtime